src/lib/bootloader/bootloader_factory.rb

```diff
@@ -135,13 +135,11 @@ def proposed_name
           return preferred_bootloader
         end
 
-        devicegraph = Y2Storage::StorageManager.instance.staging
```

Comment:
good, this removal really helps as it cause some unit tests failures in other modules.